docs/plugin_auth_keycloak.md

```diff
@@ -0,0 +1,6 @@
+**Proposed Oath2-inspired Protocol**
+Currently, we are implementing [Standard Authorization Code Flow](https://auth0.com/docs/get-started/authentication-and-authorization-flow/authorization-code-flow):
```

Comment:
We should keep all the images locally, to better control them. Let's make a copy of this image and put it in `docs/rsrc` directory. Then, you can put a reference to a web page where we got it from e.g. [img source](https://auth0.com/docs/get-started/authentication-and-authorization-flow/authorization-code-flow)